Shared/Samples/Show device location using indoor positioning/ShowDeviceLocationUsingIndoorPositioningView.swift

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import CoreLocation
+import SwiftUI
+
+struct ShowDeviceLocationUsingIndoorPositioningView: View {
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    /// This is the published value of the data that is displayed.
+    @State private var labelTextLeading: String = ""
+    /// This is the published value of the data that is displayed.
+    @State private var labelTextTrailing: String = ""
```

Comment:
User-visible text should not be calculated ahead of time as the underlying data may not the only thing that affects what is displayed to the user. For formatted values, the value may be re-formatted due to a locale change as the user moves across a border. Please re-factor to calculate the text to display to the user where it is needed. Instead of concatenating strings together with newlines, just use separate Text values in an H stack or V stack (whichever is appropriate). Instead of calling a `formatted` method on the value to be formatted, allow SwiftUI to do the formatting so that it can efficiently format the value only when needed. For example: `Text("Accuracy: \(accuracyMeasurement, format: ...)")`.